### PR TITLE
[v0.20.x] release 0.20.1

### PR DESCRIPTION
**Pull request by shouples**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this extension will be documented in this file.
 
 ## Unreleased
 
+## 0.20.1
+
 ### Fixed
```
Comment (rohitsanj) [inline CHANGELOG.md:9]: Can we add a line about https://github.com/confluentinc/vscode/pull/479 please?

**Review by rohitsanj — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rohitsanj) [reply CHANGELOG.md:9]: I'll suggest: "Fixed bug where switching between consume tabs would reset consume mode to default"

Comment (shouples) [reply CHANGELOG.md:9]: Ah I just saw that suggestion - does https://github.com/confluentinc/vscode/pull/481/commits/d9387e133850655515244b962d6d66e5718a3f48 read okay?

Comment (rohitsanj) [reply CHANGELOG.md:9]: Yeah, that's good, TYVM!

**Review by rohitsanj — APPROVED** [review]

**Pull request merged**